src/poprox_storage/repositories/accounts.py

```diff
@@ -93,11 +93,11 @@ def fetch_unassigned_accounts(self, start_date: date, end_date: date):
         group_ids = self._id_query(group_query)
 
         # Find the users allocated to those groups
-        allocation_query = select(allocation_tbl.c.account_id).where(allocation_tbl.c.group_id.in_(group_ids))
-        allocation_account_ids = self._id_query(allocation_query)
+        assignment_query = select(assignment_tbl.c.account_id).where(assignment_tbl.c.group_id.in_(group_ids))
```

Comment:
I'm pretty sure we made this change a while ago. Maybe we're just not using this function?

Reply:
Yeah, that's exactly it—we haven't tried to do anything with experiment assignments since the change happened